### ISSUE TITLE
Cut down on the number of tournaments that appear on the home page

### ISSUE THREAD

**Issue by smayya337** [Duosmium/duosmium#6]
Currently, `middleman` needs to generate previews for all 249 tournaments on the site. This takes a long time (~60% of total build time). Cutting that number down to, say, 12 would potentially decrease the build time by up to 50%.

**Comment by smayya337** [comment]
Fixed, closing.

**Comment by smayya337** [comment]
Ugh, looks like search is tied to the events that appear on the home page...

**Comment by alchzh** [comment]
I doubt its sustainable to keep everything as pre-built static content -- there's going to have to be some sort of database - fetch - display system eventually. If you want to not host a backend look into moving your data to Firebase and pulling from there

**Comment by smayya337** [comment]
@alchzh yeah, you're right - I've been working on [using Rails](https://github.com/duosmium/duosmium-rails) for database-fetch-display, but the problem is no matter how you build, the cards are time-consuming (~100ms per card -> upwards of 45s to load the home page)... there needs to be a better way to work with those cards.
